### PR TITLE
Remove support of Che4 workspace format

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Remove support of Che4 workspace format

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7826

#### Release Notes
Remove support of Che4 workspace format

#### Docs PR
n/a

### CODE REVIEW

**Review by garagatyi — APPROVED** [review]

File: wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceConfigJsonAdapterTest.java [hunk 1]
```diff
@@ -1,163 +0,0 @@
...
-
-/**
- * Tests of {@link WorkspaceConfigJsonAdapter}.
- *
- * @author Yevhenii Voevodin
- */
-@Listeners(MockitoTestNGListener.class)
-public class WorkspaceConfigJsonAdapterTest {
-
-  private static final String INVALID_CONFIGS_DIR_NAME = "invalid_configs";
```
Comment (sleshchenko) [inline wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceConfigJsonAdapterTest.java:41]: Should you remove [these resources](https://github.com/eclipse/che/tree/removeche4support/wsmaster/che-core-api-workspace/src/test/resources/invalid_configs) too or they are still used by another tests?

Comment (skabashnyuk) [reply wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceConfigJsonAdapterTest.java:41]: done

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
ci-test

**Review by vparfonov — APPROVED** [review]

**Pull request merged**